packages/perseus/src/util/parse-perseus-json/perseus-parsers/widgets-map.ts

```diff
@@ -175,6 +175,10 @@ const parseWidgetsMapEntry: (
             return parseAndAssign(`table ${id}`, parseTableWidget);
         case "video":
             return parseAndAssign(`video ${id}`, parseVideoWidget);
+        case "sequence":
+            // sequence is a deprecated widget type, and the corresponding
+            // widget component no longer exists.
+            return ctx.success(null);
```

Comment:
This will cause the sequence widget options not to be added to the `widgets` map. I should confirm that content with sequence widgets will still render the deprecated widget placeholder if those options aren't present. We might need to add a placeholder widget to the `widgets` map here in place of the sequence widget to prevent the renderer from crashing.